src/app/pages/charging-stations/registration-tokens/registration-token/charging-stations-registration-token.component.ts

```diff
@@ -128,7 +129,13 @@ export class ChargingStationsRegistrationTokenComponent implements OnInit {
       }
     }, (error) => {
       this.spinnerService.hide();
-      Utils.handleHttpError(error, this.router, this.messageService, this.centralServerService, 'tenants.create_error');
+      switch (error.status) {
+        case HTTPAuthError.FORBIDDEN:
```

Comment:
Normally if it's forbidden you should let the Utils.handleHttpError() display the message.

Reply:
what if we want to display a personnalized message letting the user know it misses the site ? because otherwise he will see something like "you cannot do this" without knowing this is only a matter of missing param ?

Reply:
> what if we want to display a personnalized message letting the user know it misses the site ? because otherwise he will see something like "you cannot do this" without knowing this is only a matter of missing param ?

You should not have a personalised message as the action is forbidden and you should not even see the action itself ;-)